lua/neogit/status.lua

```diff
@@ -859,6 +859,46 @@ local discard = function()
   vim.cmd("checktime")
 end
 
+local init_repo = function()
```

Comment:
I think this would fit better in `neogit.lib.git.init` so that the status buffer isn't exporting a git operation